tools/istio-iptables/main.go

```diff
@@ -0,0 +1,538 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net"
+	"os"
+	"os/exec"
+	"os/user"
+	"strings"
+	"time"
+
+	"github.com/joho/godotenv"
+)
+
+func getEnvWithDefault(key string, defaultValue string) string {
+	value := os.Getenv(key)
```

Comment:
@geeknoid we have a linter rule that blocks os.Getenv. Not sure why it isn't running here, maybe it doesn't run on tools. Along those lines, tools may not be the best place for this -- this is first class Istio code not just a tool.